go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -65,21 +79,24 @@ func Init(sandboxName string,
 	// Create new active scenario model
 	modelCfg := mod.ModelCfg{
 		Name:      "activeScenario",
-		Namespace: sandboxName,
+		Namespace: sbi.sandboxName,
 		Module:    moduleName,
 		UpdateCb:  nil,
-		DbAddr:    redisAddr,
+		DbAddr:    cfg.RedisAddr,
 	}
 	sbi.activeModel, err = mod.NewModel(modelCfg)
 	if err != nil {
 		log.Error("Failed to create model: ", err.Error())
 		return err
 	}
 
-	sbi.updateUeEcgiInfoCB = updateUeEcgiInfo
-	sbi.updateAppEcgiInfoCB = updateAppEcgiInfo
-	sbi.updateScenarioNameCB = updateScenarioName
-	sbi.cleanUpCB = cleanUp
+	// Connect to Postgis DB
+	sbi.pc, err = postgis.NewConnector(geModuleName, sbi.sandboxName, postgisUser, postgisPwd, cfg.PostgisHost, cfg.PostgisPort)
```

Comment:
Should a high level service such as Location connect directly to the geospatial database?
Was expecting that it would use the gis package that hides details about the database.